src/views/dashboard/components/header/header.scss

```diff
@@ -1,168 +1,260 @@
-@import '../../../../assets/styles/global/colors.scss';
-@import '../../../../assets/styles/global/breakpoint.scss';
+@import "../../../../assets/styles/global/colors.scss";
+@import "../../../../assets/styles/global/breakpoint.scss";
 
 .menu {
-    background: $p-color-white;
-    color: $p-color-black;
-    position: relative;
+  background: $p-color-white;
+  color: $p-color-black;
+  position: relative;
 
-    .logo {
-        padding: 2rem 6rem;
-        height: 8rem;
+  .logo {
+    padding: 2rem 6rem;
+    height: 8rem;
 
-        @media only screen and (max-width: $bp-medium) {
-            height: 6rem;
-            padding: 2rem;
-        }
+    @media only screen and (max-width: $bp-xlarge) {
+      padding: 2rem 3rem;
     }
+  }
+
+  &-box {
+    width: 100%;
+  }
+
+  &-nav {
+    padding: 0 !important;
+    height: 100%;
+    font-size: 2.5rem;
+    list-style: none;
 
-    &-box {
-        width: 100%;
+    @media only screen and (max-width: $bp-xlarge) {
+      display: none !important;
```

Comment:
do we need important?

Reply:
yes, it's not applied, probably because we also have 'p-flex' class on this element ('display' property overlapping)